apollo/resolvers/workspace/updateWorkspace.ts

```diff
@@ -0,0 +1,24 @@
+import { GQLContext } from '@/apollo/interfaces';
+
+interface Args {
+  id: string;
+  name: string;
+}
+
+export const updateWorkspace = async (
+  _: Record<string, never>,
+  { id, name }: Args,
+  context: GQLContext,
+) => {
+  const { db, session } = context;
+  const workspace = await db.workspaceRepo.get(id);
+
+  if (!workspace?.users.map(String).includes(session.user.id)) {
+    return null;
+  }
```

Comment:
Shouldn't this be an error response if the user ends up in a scenario where they attempt to update a workspace that they don't exist on?